detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ForbiddenMethodCall.kt

```diff
@@ -70,6 +70,7 @@ class ForbiddenMethodCall(config: Config) : Rule(
                 "`String.toBigDecimalOrNull()` instead.",
             "java.math.BigDecimal.<init>(kotlin.String)" to "using `BigDecimal(String)` can result in a " +
                 "`NumberFormatException`. Use `String.toBigDecimalOrNull()`",
+            "kotlin.system.measureTimeMillis" to "It is deprecated since 1.9.",
```

Comment:
```suggestion
            "kotlin.system.measureTimeMillis" to "It is marked as obsolete. Use `kotlin.time.measureTime` instead.",
```

As far as I can see it's not deprecated but they say in the documentation that it's obsolete. I'm not sure why they don't marked as `@Deprecated` too...